.travis.yml

```diff
@@ -1,82 +1,50 @@
 sudo: true
-language: generic
+language: haskell
 
 git:
   depth: 5
 
+cabal: "head"
+
 cache:
   directories:
   - "$HOME/.cabal"
+  - "$HOME/.cabal/store"
```

Comment:
I think `.cabal/store` can be used instead of `.cabal`. Looks like it's enough to cache only `store` subdirectory of `.cabal` folder. I'm not aware of full content for `.cabal` directory though...